src/content/quickstart/openshift/create_clusters.md

```diff
@@ -0,0 +1,53 @@
+### Create and deploy cluster A
+
+In this step you will deploy cluster A, with the default IP CIDRs
```

Comment:
Remember the PR was talking about taking the common bits from here: https://github.com/submariner-io/submariner-website/commit/9fa6eee454c719daa01d0cbff63302779da5d7d4

Can you update this text (and other common texts from .. src/content/quickstart/openshiftgn/_index.md ) it has more review and fixes (at least the phrasings, ... 

Be aware of the cmdline and IP ranges which are different.